mmv1/products/firebase/api.yaml

```diff
@@ -57,16 +57,77 @@ objects:
           'https://firebase.google.com/'
       api: 'https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects'
     properties:
-      - !ruby/object:Api::Type::String
+      - !ruby/object:Api::Type::Integer
         name: projectNumber
         output: true
         description: |
-          The number of the google project that firebase is enabled on.
+          Immutable. The globally unique, Google-assigned canonical identifier
+          for the Project. Use this identifier when configuring integrations and/or
+          making API calls to Firebase or third-party services.
+      - !ruby/object:Api::Type::String
```

Comment:
it would be uncommon for a developer to have a FirebaseProject in their .tf file and not have the parent Cloud Project.  I suspect we should omit Project Id as [this](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project#project_id) is the preferred place to read it.

Reply:
Done.